caches.json

```diff
@@ -0,0 +1,9 @@
+{
+  "caches": [
+    {
+      "id": "google",
+      "name": "Google AMP Cache",
+      "updatePingApiUrlPrefix": "https://cdn.ampproject.org/"
+    }
+  ]
+}
```

Comment:
nit: Some json files in the code base have a trailing newline while others don't. Not sure what the preferred choice is here.